src/directLine.ts

```diff
@@ -252,6 +261,7 @@ export interface IBotConnection {
     connectionStatus$: BehaviorSubject<ConnectionStatus>,
     activity$: Observable<Activity>,
     end(): void,
+    getConversation(): Conversation,
```

Comment:
Well I guess that gets to the next point, which is that WebChat also shouldn't have any dependency on the deets of its botConnection. I currently have a second implementation of IBotConnection for browser-based bots and there is no such thing as a Conversation.

So I guess the question is, why does WebChat suddenly need to know about a Conversation when it has gone this long without one?